api/apps/v1alpha1/nimcache_types.go

```diff
@@ -40,6 +41,8 @@ type NIMCacheSpec struct {
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
 	// NodeSelectors are the node selector labels to schedule the caching job.
 	NodeSelectors map[string]string `json:"gpuSelectors,omitempty"`
+	UserId        *int64            `json:"userId,omitempty"`
```

Comment:
Nit : change to UserID / groupID -- Not sure why not reported but some golint rules may complain; same for the getter function names

Reply:
Updated it